projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.html

```diff
@@ -0,0 +1,96 @@
+<igx-input-group #inputGroup [displayDensity]="displayDensity" [type]="type">
+    <ng-container ngProjectAs="[igxLabel]">
+        <ng-content select="[igxLabel]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-prefix">
+        <ng-content select="igx-prefix"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-hint, [igxHint]">
+        <ng-content select="igx-hint, [igxHint]"></ng-content>
+    </ng-container>
+
+    <input #comboInput igxInput [value]="value" (input)="handleInputChange($event)" (keyup)="handleKeyUp($event)"
+        (keydown)="handleKeyDown($event)" (blur)="onBlur()" [attr.placeholder]="placeholder || searchPlaceholder"
```

Comment:
either remove `placeholder` or `searchPlaceholder`. Having both these is contradictory.